frontend/public/components/_sysevent-stream.scss

```diff
@@ -182,4 +182,8 @@ $color-dark-border: #ddd;
   .co-sysevent-stream__timeline__end-message {
     left: 45px;
   }
+
+  .co-icon-and-text {
+    display: inline !important;
```

Comment:
I'm unclear why this is needed

Reply:
when I use <StatusIcon status={phase} />  it returns icon with actual status and fills to entire row in a column so I can't place the PV or PVC name in the same row, it displayed in next row.
![image](https://user-images.githubusercontent.com/15073590/54014380-754d0500-41a2-11e9-8e32-1f98ea469e4e.png)


To display status with an icon and any other text next to that I used the display inline

Reply:
But won't this affect a bunch of other places where the class is used? I'd think we'd want a specific BEM modifier if we need this behavior. Also this is definitely not the right file for this change :)

I'd still prefer moving the PVC to a separate column. Then we don't have the issue.

Reply:
I have verified it is not affecting other place but still I am agreed your point to move PVC to separate  column  because when PVC name is big then it won't look good because we are displaying too many details in a same column

Reply:
I have modified CSS logic in a correct file: https://github.com/openshift/console/pull/1261/files#diff-b9d777b57b1021f2311b7e34bdd40469R3